Dockerfile

```diff
@@ -1,34 +1,34 @@
-FROM openjdk:8
+# Base image used for both building and running
+FROM openjdk:8-jre-alpine AS base
 
 ENV GEOSERVER_VERSION 2.14.3
 ENV GEOSERVER_VERSION_MM 2.14
-ENV GEOSERVER_HOME /usr/local/geoserver
+ENV GEOSERVER_HOME /geoserver-${GEOSERVER_VERSION}
 ENV GEOSERVER_DATA_DIR /var/geoserver/data
 
-RUN apt-get update && apt-get install -y gettext
 RUN mkdir -p ${GEOSERVER_HOME}
-RUN mkdir -p /var/geoserver
+RUN mkdir -p ${GEOSERVER_DATA_DIR}
 
-RUN cd /tmp && \
-    curl -L -O http://sourceforge.net/projects/geoserver/files/GeoServer/${GEOSERVER_VERSION}/geoserver-${GEOSERVER_VERSION}-bin.zip && \
-    unzip geoserver-${GEOSERVER_VERSION}-bin.zip && \
-    mv geoserver-${GEOSERVER_VERSION}/* ${GEOSERVER_HOME}
 
-RUN mv ${GEOSERVER_HOME}/data_dir ${GEOSERVER_DATA_DIR}
-RUN cd /tmp && \
-    curl -OL https://build.geoserver.org/geoserver/${GEOSERVER_VERSION_MM}.x/community-latest/geoserver-${GEOSERVER_VERSION_MM}-SNAPSHOT-s3-geotiff-plugin.zip && \
+# Build image
+FROM base AS build
+
+RUN apk add --no-cache curl
+RUN curl -OL http://sourceforge.net/projects/geoserver/files/GeoServer/${GEOSERVER_VERSION}/geoserver-${GEOSERVER_VERSION}-bin.zip && \
```

Comment:
🙈

Reply:
This is the link given on the GeoServer website, which is dumb that they don't provide the https link. I just checked though, and it redirects to https. I'll go ahead and change it just because.

Reply:
I was 🙈 about sourceforge being a thing in 2019.